compiler/parser/src/function.rs

```diff
@@ -1,16 +1,21 @@
+// Contains functions that perform validation and parsing of arguments and parameters.
+// Checks apply both to functions and to lambdas.
```

Comment:
Did you intend doc-comment for module?

```suggestion
//! Contains functions that perform validation and parsing of arguments and parameters.
//! Checks apply both to functions and to lambdas.
```

Reply:
yeah since it isn't exposed atm and I don't think we'll ever expose it (since its an internal parser detail).